backend/benefit/companies/models.py

```diff
@@ -0,0 +1,13 @@
+from django.utils.translation import gettext_lazy as _
+from localflavor.generic.models import IBANField
+
+from shared.models.abstract_models import AbstractCompany
+
+
+class Company(AbstractCompany):
+    bank_account_number = IBANField(
+        include_countries=("FI",), verbose_name=_("bank account number")
+    )
```

Comment:
Are all fields here that we discussed in the sprint planning?

Reply:
Yeah I think that pretty much all we need. The rest of the fields are inherited from the AbstractCompany fields